### PR TITLE
setup bintray publication

### PR DESCRIPTION
**Pull request by dsvoronin**
will address #1 

- [x] applying instructions in readme
- [x] sync with jcenter
- [x] refactor kts mess

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
-# ui-test-lib
+# Avito android UI testing library
```
Comment (lukaville) [inline README.md]: maybe `android` -> `Android`?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
 
 ## Should i use it
```
Comment (lukaville) [inline README.md]: `i` -> `I`

File: build.gradle.kts [hunk 3]
```diff
@@ -1,23 +1,128 @@
...
+import com.android.build.gradle.BaseExtension
+import com.android.build.gradle.LibraryPlugin
+import com.android.builder.model.AndroidLibrary
+import com.jfrog.bintray.gradle.BintrayExtension
+import org.gradle.api.internal.plugins.DslObject
+import org.gradle.api.publish.maven.MavenPom
+import kotlin.properties.Delegates
+
 buildscript {
+    val kotlinVersion:String by project
```
Comment (lukaville) [inline build.gradle.kts]: add space between `val kotlinVersion:` and `String`

**Review by lukaville — DISMISSED** [review]

**Review by lukaville — APPROVED** [review]

File: build.gradle.kts [hunk 4]
```diff
@@ -1,23 +1,128 @@
...
 
-subprojects {
-    repositories {
+group = "com.avito.ui-testing"
+version = "0.1.0"
+
+val minSdk: String by project
+val targetSdk: String by project
+
+fun Project.setupAndroidExtension() {
```
Comment (dimorinny) [inline build.gradle.kts:33]: Let's write it on every module with injecting `minSdk` and `targetSdk`. Now it looks a little bit confusing for me

**Pull request merged**